src/FixedDataTableBufferedRows.js

```diff
@@ -142,7 +142,13 @@ class FixedDataTableBufferedRows extends React.Component {
     // We translate all the rows together with a parent div. This saves a lot of renders.
     const style = {};
     FixedDataTableTranslateDOMPosition(style, 0, containerOffsetTop, false);
-    return <div style={style}>{this._staticRowArray}</div>;
+
+    // NOTE (pradeep): Sort the rows by row index so that they appear with the right order in the DOM (see #221)
+    const sortedRows = _.sortBy(this._staticRowArray, (row) =>
```

Comment:
`this._staticRowArray` is not ordered by row Index, as seen [here](https://github.com/schrodinger/fixed-data-table-2/blob/6efc3c620c19f174329e18b9e57b5a67461c5a03/src/FixedDataTableBufferedRows.js#L135-L136).
Here, I make `sortedRows` which is just the sorted version of `this._staticRowArray`, ordered by row index.

Reply:
had some issues with selenium tests and row indexes (our tests assumed DOM order is the display order), so I tried this fix as well. 

I was able to workaround the test case issues by sorting by 'props.ariaRowIndex' instead of 'props.index'. 

Would it be better to 'props.ariaRowIndex' here to ensure correct DOM order vs. the display order?

Reply:
@eriihine , thanks for the suggestion. 

`props.ariaRowIndex` is accurate for rows in context of the whole table (because it also consider headers, footers, etc), where as `props.index` is only accurate within the body rows.

I think this doesn't make a difference _right now_ because the jumbled up rows are just the body rows, and are only within the `FixedDataTableBufferedRows.js` component. 
So our usage here should be safe, but using `ariaRowIndex` definitely seems safer.

I'll make the change, thanks.

Reply:
@pradeepnschrodinger great, would it possible to consider to include this fix to some future v1.2.x release?

Reply:
@eriihine , sure.
I'll backport the fix to v1.2.x as soon as this gets merged.

Reply:
this would be great, thanks!